packages/common/src/pages/defaults.ts

```diff
@@ -0,0 +1,48 @@
+import { IHubPage } from "../core";
+import { IModel } from "../types";
+
+export const HUB_PAGE_ITEM_TYPE = "Hub Page";
+export const ENTERPRISE_PAGE_ITEM_TYPE = "Site Page";
+export const PAGE_TYPE_KEYWORD = "hubPage";
+/**
+ * Default values of a IHubPage
+ */
+export const DEFAULT_PAGE: Partial<IHubPage> = {
+  name: "No title provided",
+  permissions: [],
+  schemaVersion: 1,
+  tags: [],
+  typeKeywords: [PAGE_TYPE_KEYWORD],
+  view: {
```

Comment:
Do we want to have a default layout - put another way - should a freshly created page be "renderable"?